mlir/utils/jenkins/Jenkinsfile

```diff
@@ -53,11 +53,8 @@ void buildMIGraphX(String cmakeOpts) {
 }
 
 void getAndBuildMIGraphX(String cmakeOpts) {
-    // TODO: https://github.com/ROCmSoftwarePlatform/llvm-project-private/issues/889
-    // git branch: params.MIGraphXBranch, poll: false,\
-    //     url: 'https://github.com/ROCmSoftwarePlatform/AMDMIGraphX.git'
-    checkout([$class: 'GitSCM', branches: [[name: 'mlir-fix-dot-format' ]],
-              userRemoteConfigs: [[url: 'https://github.com/ROCmSoftwarePlatform/AMDMIGraphX.git']]])
+    git branch: params.MIGraphXBranch, poll: false,\
```

Comment:
... I'll note that this doesn't actually change the branch to `develop`, but instead makes the branch my upstream branch PR.